 core: copy fstab before invoking ota_from_target_files

That file is needed to manage partitions etc.

diff --git a/core/Makefile b/core/Makefile
index 43c98e3..b227300 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -2774,6 +2774,9 @@ endif
 $(INTERNAL_OTA_PACKAGE_TARGET): $(BUILT_TARGET_FILES_PACKAGE) \
 		build/tools/releasetools/ota_from_target_files
 	@echo "Package OTA: $@"
+	$(hide) mkdir -p $(patsubst %.zip,%,$(BUILT_TARGET_FILES_PACKAGE))/RECOVERY/RAMDISK/etc/
+	$(hide) cp $(recovery_fstab) \
+		$(patsubst %.zip,%,$(BUILT_TARGET_FILES_PACKAGE))/RECOVERY/RAMDISK/etc/recovery.fstab
 	$(hide) PATH=$(foreach p,$(INTERNAL_USERIMAGES_BINARY_PATHS),$(p):)$$PATH MKBOOTIMG=$(MKBOOTIMG) \
 	   ./build/tools/releasetools/ota_from_target_files -v \
 	   --block \
